Articles/Language-Files/extending.md

```diff
@@ -0,0 +1,135 @@
+# Language Files & Localization
+
+Language files are used to translate the Umbraco backoffice user interface so that end users can use Umbraco in their native language. This is particularly important for content editors who do not speak English.
+
+If you are a package developer, [see here for docs on how to include translations for your own package](Language-Files-For-Packages/index.md).
+
+## Supported Languages
+Current languages that are included in the core are:
+
+- English (UK)
+- English (US)
+- Danish
+- German
+- Spanish
+- French
+- Hebrew (Israel)
+- Italian
+- Japanese
+- Korean
+- Dutch
+- Norwegian
+- Polish
+- Portuguese
+- Russian
+- Swedish
+- Chinese
+- Chinese (Taiwan)
+- Czech
+- Turkish
+
+## Where to find the language files
+
+### Core language files
+The core Umbraco language files are found at the following location within the Umbraco source:
+
+    Umbraco-CMS/src/Umbraco.Web.UI.Netcore/umbraco/config/lang/
+
+These language files are the ones shipped with Umbraco and should not be modified.
+
+### Package language files
+If you are a package developer, [see here for docs on how to include translations for your own package](Language-Files-For-Packages/index.md), package language files are located in:
+
+    /App_Plugins/mypackage/lang/{language}.xml
+
+### User language files
+If you want to override Umbraco core translations or translations shipped with packages, you can do that too, these files are located here:
+
+    /config/lang/{language}.user.xml
+
+By default, these files are empty but you can add any new keys you want or override existing ones with your own translations. The nice part about the user files is that they will not get overwritten by the installer when you upgrade your Umbraco versions.
+
+## Using the language keys
+Using core or custom language keys from your code:
+
+
+### From .NET
+`ILocalizedTextService` is is used to localize strings, and is available through dependency injection. First inject the service, and then use the Localize method available in the namespace `Umbraco.Extensions` to localize the string with the format [area]/[key]:
```

Comment:
```suggestion
`ILocalizedTextService` is used to localize strings, and is available through dependency injection. First, inject the service, and then use the `Localize()` method available in the namespace `Umbraco.Extensions` to localize the string with the format [area]/[key]:
```

Reply:
Thanks! Quite the oopsie with the double is 😄